### DISCUSSION TITLE
Scan result summary

### DISCUSSION THREAD

**Discussion by tblinux** [terraform-linters/tflint#1758] [Ideas]
## Introduction
Scan report summary becomes important feature when measure the project's code quality. `tflint` does support the different formats in which we can generate report but detailed summary part seems to be missing. 

## Proposal
At the end of scan, tflint should provide detailed summary of results. For example:

`tflint scanned 19 templates against 78 rules and found 12 errors, 78 warnings, and 0 notice violations. `

**Comment by bendrucker** [comment]
Can you cite prior art, ie other tools that offer a similar summary?

**Comment by tblinux** [comment]
Yes, Actually Idea came from the other tool only. `cfn-lint` does provide similar summary.

Command: `cfn-lint -f pretty *.yaml`

Summary: `Cfn-lint scanned 98 templates against 102 rules and found 43 errors, 321 warnings, and 0 informational violations`

**Reply by PatMyron** [reply]
https://github.com/aws-cloudformation/cfn-lint/issues/882, https://github.com/aws-cloudformation/cfn-lint/pull/1742

**Comment by wata727** [comment]
This idea is tracked in https://github.com/terraform-linters/tflint/issues/1849
